### PR TITLE
[gps] fix ubx ucenter autobaud and autoconf

### LINKED ISSUE #1459: UBLOX 8 autoconfiguration
- NAV5 message has bits used in a different way

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/28809050-ublox-8-autoconfiguration?utm_campaign=plugin&utm_content=tracker%2F10895&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F10895&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by gautierhattenberger**
should fix #1459 
tested on UBX M8M module with default config at startup

### CODE REVIEW

File: sw/airborne/modules/gps/gps_ubx_ucenter.c [hunk 1]
```diff
@@ -457,12 +461,12 @@ static bool_t gps_ubx_ucenter_configure(uint8_t nr)
       if (gps_ubx_ucenter.reply != GPS_UBX_UCENTER_REPLY_ACK) {
         DEBUG_PRINT("ublox did not acknowledge port configuration.\n");
       } else {
-        DEBUG_PRINT("Changed ublox baudrate to: %u\n", UART_SPEED(UBX_GPS_BAUD));
+        DEBUG_PRINT("Changed ublox baudrate to: %u\n", UART_BAUD(gps_ubx_ucenter.baud_target));
```
Comment (flixr) [inline sw/airborne/modules/gps/gps_ubx_ucenter.c]: should be UART_SPEED?

**Comment by gautierhattenberger** [comment]
Should be fine now. @fvantienen can you test on Bebop2 or any other UBX8 gps ?

**Comment by flixr** [comment]
@dewagter maybe you can test this, since @fvantienen is on vacation?

**Comment by gautierhattenberger** [comment]
Actually this thing is half broken since f4ada86a90f864904b25d52d6a9dd79042897d90. Since it is saving the conf at the end and that that most of the receiver have flash, any older gps using ucenter module kept working because parameters were already correct.
I'm on vacation now, and probably most people at Delft as well. Since it is working on Ublox 8 and certainly not correctly on other modules, I suggest to merge this so the next release is not blocked by this.

**Comment by dewagter** [comment]
I can confirm it now works nicely: change from any baudrate to any baudrate, also on ublox8 devices.

**Pull request merged**